### PR TITLE
Apply "width" token to Switch root

### PR DESCRIPTION
**Pull request by lawrencewin**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

In re-testing Switch tokens to try and publish it as V1, I found that the "width" token was not being applied. This PR applies the token to the root while cleaning up the existing code. (minWidth and minHeight get applied twice - edited to be applied once)

### Verification

Tested in FURN tester. Below is the CustomizedSwitch section with width set to 500px. 

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| image | image |

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [ ] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: packages/experimental/Switch/src/Switch.styling.ts [hunk 1]
```diff
@@ -34,24 +34,14 @@ export const stylingSettings: UseStylingOptions<SwitchProps, SwitchSlotProps, Sw
           alignItems: 'center',
           flexDirection: tokens.flexDirection,
           alignSelf: 'flex-start',
-          minHeight: tokens.minHeight,
-          minWidth: tokens.minWidth,
+          width: tokens.width,
           borderColor: tokens.focusStrokeColor,
           borderWidth: tokens.focusBorderWidth,
           borderRadius: tokens.focusBorderRadius,
           ...layoutStyles.from(tokens, theme),
```
Comment (Saadnajmi) [inline packages/experimental/Switch/src/Switch.styling.ts:41]: Shouldn't these be part of `...layoutStyles` ?

Comment (lawrencewin) [reply packages/experimental/Switch/src/Switch.styling.ts:41]: minWidth, and minHeight are. Width currently isn't, but it probably should be.

Comment (rurikoaraki) [reply packages/experimental/Switch/src/Switch.styling.ts:40]: Yeah you can add it. It's mostly a list of tokens that have manually been added over time, there isn't really a reason why a particular token has been left out.

**Review by rurikoaraki — APPROVED** [review]

File: packages/utils/tokens/src/layout-tokens.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ import type { OperationSet } from './token.types';
 import { tokenBuilder } from './tokenBuilder';
 
 export interface LayoutTokens {
+  width?: ViewStyle['width'];
```
Comment (Saadnajmi) [inline packages/utils/tokens/src/layout-tokens.ts:9]: should we add height as well? Also, did we confirm that no other set of tokens uses width/height?

**Review by Saadnajmi — APPROVED** [review]

File: packages/utils/tokens/src/layout-tokens.ts (hunk 2 above)
Comment (Saadnajmi) [reply packages/utils/tokens/src/layout-tokens.ts:9]: OK I don't see any other token sets that use width/height. But I would prefer we _also_ add height.

**Pull request merged**